### PR TITLE
configuration changes for hypopedia and scratchpad

### PR DESCRIPTION
**Pull request by Ultracane1997**
Making use of $wgRestrictionLevels, $wgRevokePermissions, $wgManageWikiPermissionsAdditionalRights, $wgManageWikiPermissionsAdditionalAddGroups, and $wgManageWikiPermissionsAdditionalRemoveGroups for my wiki projects. I'm trying to add the bureaucrat, extendedconfirmed, and leader restriction levels for both Hypopedia and Scratchpad. I'm also adding a banned user group with $wgRevokePermissions to Hypopedia and Scratchpad.

### CODE REVIEW

**Comment by Ultracane1997** [comment]
I also removed the configuration variables for cyclonepediawiki as it no longer exists.

**Review by dmehus — COMMENTED** [review]
This looks fine to me, based on my initial look. I encouraged @Ultracane1997 to just do the GitHub PR himself for, since he is familiar with editing the `LocalSettings.php` file for MediaWiki, rather than doing the Phabricator request first.

**Comment by Ultracane1997** [comment]
I just got informed that editing the steward user group by myself is not allowed, so if that is indeed the case I could just remove that part and submit the rest of the changes.

File: LocalSettings.php [hunk 1]
```diff
@@ -2115,6 +2114,36 @@
 			'bureaucrat' => [
 				'bureaucrat' => true,
 			],
+			'extendedconfirmed' => [
+				'extendedconfirmed' => true,
+			],
+			'leader' => [
+				'leader' => true,
+			],
+			'steward' => [
```
Comment (Universal-Omega) [inline LocalSettings.php:2123]: Just wondering what you're doing modifying the steward usergroup locally? Seem to be adding oversight and CheckUser into Steward. I'm not certain, but that should be discouraged. Stewards can add themselves to those individual usergroups when really necessary.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (Ultracane1997) [reply LocalSettings.php:2123]: Well, I will just end up not doing it and remove that part of the changes, but I will keep the rest of the changes.

Comment (Ultracane1997) [reply LocalSettings.php]: I have now updated it so that's no longer happening.

**Comment by Ultracane1997** [comment]
If it's any easier to note this I have removed the local steward user group modification in the second commit.

**Review by Universal-Omega — APPROVED** [review]
All looks good to me now.

**Pull request closed without merging**